### PR TITLE
fix(slider): update slider states and documentation

### PR DESCRIPTION
**Pull request by varnikajain15**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

**Review by Rashi-Gupta-2000 — COMMENTED** [review]
Please change the commit message and title to `states`

**Review by Rashi-Gupta-2000 — COMMENTED** [review]
Please check the states
The color of the dash above number should be black in case of selected
Screenshot_2023-01-18_13-56-28

**Review by Rashi-Gupta-2000 — COMMENTED** [review]
The image inserted for active slider needs to be changed from design point please check with the designers

**Review by Rashi-Gupta-2000 — COMMENTED** [review]
Please check with the designers if there should be a focus ring when the knob is in active state

**Review by Rashi-Gupta-2000 — DISMISSED** [review]

File: core/components/atoms/multiSlider/index.tsx [hunk 1]
```diff
@@ -334,7 +334,7 @@ export class MultiSlider extends React.Component<InternalMultiSliderProps, Multi
           data-test="DesignSystem-MultiSlider-Label"
         >
           {/* eslint-enable  */}
-          <span className={'Slider-ticks'} />
+          <span className={`Slider-ticks ${active && 'Slider-ticks--active'}`} />
```
Comment (anuradha9712) [inline core/components/atoms/multiSlider/index.tsx]: Please use a `classNames` to define a classes on conditional basis

File: core/components/atoms/slider/__tests__/Slider.test.tsx [hunk 2]
```diff
@@ -40,3 +40,16 @@ describe('Slider component', () => {
...
+  it('check for onChange function call', () => {
+    const onChangeHandler = jest.fn();
+    const { getByTestId } = render(<Slider label="Slider Label" onChange={onChangeHandler} />);
+    const sliderHandle = getByTestId('DesignSystem-MultiSlider-Handle');
+    fireEvent.click(sliderHandle);
+    fireEvent.keyUp(sliderHandle, { keyCode: 37 });
+    fireEvent.keyDown(sliderHandle, { keyCode: 37 });
+    fireEvent.keyDown(sliderHandle, { keyCode: 39 });
+    expect(onChangeHandler).toBeCalled();
+  });
```
Comment (anuradha9712) [inline core/components/atoms/slider/__tests__/Slider.test.tsx:54]: Please add a test case to check if correct value is passed in `onChangeHandler` callback function

File: css/src/components/slider.css [hunk 3]
```diff
@@ -55,6 +55,10 @@
   background-color: var(--secondary-dark);
 }
 
+.Slider-ticks--active {
+  background-color: var(--inverse);
+}
+
```
Comment (anuradha9712) [inline css/src/components/slider.css]: Instead of defining new class please use existing background class `bg-dark`

**Review by anuradha9712 — APPROVED** [review]

**Pull request merged**